.appveyor.yml

```diff
@@ -21,7 +20,8 @@ install:
 - ps:  for ($i=0; $i -lt $packages.length; $i++) {
            $env:ALL_PACKAGES += $packages[$i] + ":" + $env:PLATFORM + "-windows-static "
        }
-- cmd: git -C C:\Tools\vcpkg pull # This is a temporary fix, can be removed after appveyor update its image to include Microsoft/vcpkg#4046
+- cmd: git -C C:\Tools\vcpkg fetch https://github.com/ken2812221/vcpkg.git unicode-branch
+- cmd: git -C C:\Tools\vcpkg checkout FETCH_HEAD
```

Comment:
This is just for testing that apply utf-8 filename changes, it would have to change upstream.